.gitignore

```diff
@@ -3,3 +3,5 @@ Xcode*
 *~
 *.pyc
 coverage*
+.vscode*
+_build*
```

Comment:
This line is specific to your settings. I would prefer to not put it here. @tdouin Do you agree?

Reply:
yes sure.

Reply:
@tdouin: for information, you can add eg.
```
[core]
    excludesfile = ~/.global_gitignore
```
In your `~/.gitconfig`, and put those `.vscode*` & `_build*` in your `~/.global_gitignore`, and never worry about those again :)

Reply:
Ok i get it, thanks for the advice